tests/test_everything.py

```diff
@@ -60,7 +61,7 @@ def test_fix_future_year_typo(self):
             '12/01/2806': '12/01/2806',                     # Should not change
             '12/01/2886': '12/01/2886',                     # Should not change
         }
-        for before, after in expectations.iteritems():
+        for before, after in six.iteritems(expectations):
```

Comment:
Huh, wasn't aware of that. Seems to be functionally the same although it doesn't return an iterator. I think for this case since the data isn't large it's moot. I'll back out the calls to `six.iteritems`.